crates/wash-cli/src/keys.rs

```diff
@@ -15,7 +15,7 @@ const NKEYS_EXTENSION: &str = ".nk";
 pub enum KeysCliCommand {
     #[clap(name = "gen", about = "Generates a keypair")]
     GenCommand {
-        /// The type of keypair to generate. May be Account, User, Module (or Component), Service (or Provider), Server (or Host), Operator, Cluster
+        /// The type of keypair to generate. May be Account, User, Module (or Component), Service (or Provider), Server (or Host), Operator, Cluster, X25519
```

Comment:
I believe they are called Curve in [`nk`](https://github.com/wasmCloud/nkeys/blob/main/src/bin/nk/main.rs#L26), I think using `Curve` here would be good for consistency :)

Reply:
well look at that, looks like they both are set this way. Let me add an alias at least
```
➜ nk -gen
flag needs an argument: -gen
Usage: nk [options]
    -v                    Show version
    -gen <type>           Generate key for [user|account|server|cluster|operator|curve|x25519]
```